bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/TTZH_5f_LO/TTZH_5f_LO_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define p = p b b~
+define j = j b b~
```

Comment:
these lines are not needed

Reply:
once again, checked with meng together now, line 7 is needed. You dont need t redefine the p and j.